### ISSUE TITLE
Error pages

### ISSUE THREAD

**Issue by michaeldfallen** [hmcts/one-per-page#20]
Lazy clients will want error pages that cover the standard errors, 404, 500, etc.

These should be defined by one-per-page and defaulted in the journey function but overrideable by the user:

```
journey(app, {
  ...
  errorPages: {
    500: new My500ErrorPage(),
    404: new My404ErrorPage()
  }
});
```

**Comment by dtaylor7** [comment]
resolved by #26

**Comment by michaeldfallen** [comment]
Can't see where #26 fixes this issue. Reopening.
